pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -477,6 +480,18 @@ type SnapshotScheduleSpec struct {
 	StartTime string `json:"startTime,omitempty"`
 }
 
+// QuotaSpec represents the spec for quotas in a pool
+type QuotaSpec struct {
+	// Enabled represents whether pool quotas are set
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
Do we really need this parameter? Doesn't the nonzero presence of either `MaxBytes` or `MaxObjects` imply `Enabled = true`?

Reply:
We decided to keep it in for backwards compatibility and safety . Please see this resolved conversation:

https://github.com/rook/rook/pull/7264#pullrequestreview-593573783